### PR TITLE
Add README.md

### PR DESCRIPTION
**Pull request by chase-qi**
- Update automated/README using markdown syntax
- Move the file out as repo README

Signed-off-by: Chase Qi <chase.qi@linaro.org>

### CODE REVIEW

**Comment by chase-qi** [comment]
Hi Milosz, Dan,

Would you please  review the readme file? I am asking for comments on all the aspects include typo, syntax, wrong grammar or expressions, etc.

Thanks,
Chase

**Comment by mwasilew** [comment]
@chase-qi  will this show up on git.linaro.org?

**Comment by chase-qi** [comment]
@mwasilew yes, everything added to this repo will be synced back to git.l.o/qa/test-definitions

File: README.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
+# Test Definitions
+A test suite works with and without [LAVA](http://lavasoftware.org/). The following two
+sets of automated tests are supported.
+
+- `automated/linux/`
+- `automated/android/`
+
+For each test case, both test script and corresponding test definition files provided in
+the same folder that named after test case name. Test scripts are self-contained and
```
Comment (danrue) [inline README.md]: For each test case, both the test script and the corresponding test definition files are provided in￼ the same folder and are named after the test case name.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+works independently. Test definition files in YAML format are provided for test run with
```
Comment (danrue) [inline README.md]: work*, runs* :smile_cat:

File: README.md [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+local test-runner and within LAVA.
+
+## Installation
+Installing the latest development version:
+
+    git clone https://git.linaro.org/qa/test-definitions.git
```
Comment (danrue) [inline README.md]: Update the url to github here?

File: README.md [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+    cd ./test-definitions
+    . ./automated/bin/setenv.sh
+    pip install -r ${REPO_PATH}/automated/utils/requirements.txt
```
Comment (danrue) [inline README.md]: Side note, does this repo support python3?

Comment (chase-qi) [reply README.md]: test-runner supports both python2 and python3.

**Comment by chase-qi** [comment]
@danrue thanks a lot for the review! I have fixed the issues.

**Pull request merged**